src/components/image/Image.constants.ts

```diff
@@ -0,0 +1,14 @@
+import { TShapeBorderRadius } from '../../foundations/shape';
+import exampleImg from '../../assets/images/image-example.png';
+
+import TImageSize from './Image.types';
+
+const DEFAULT_IMAGE_SIZE: TImageSize = 'auto';
+
+const DEFAULT_IMAGE_BORDER_RADIUS: TShapeBorderRadius = 4;
+
+const DEFAULT_IMAGE_SOURCE = exampleImg;
```

Comment:
I think we should just import a image in the story or request a dummy image source from the web